sources/packages/forms/src/form-definitions/supportingusersparent2022-2023.json

```diff
@@ -2659,7 +2684,11 @@
                   "properties": {},
                   "allowMultipleMasks": false,
                   "addons": [],
-                  "id": "esfhhsd"
+                  "id": "esfhhsd",
+                  "inputType": "text",
+                  "inputMask": "",
+                  "decimalLimit": 2,
```

Comment:
For this and other inputs in this file, is the `decimalLimit": 2``something new being configured beyond the income inputs?

Reply:
Need to discuss this one with you.